build-support/ci_blacklists/unit_test_v2_blacklist.txt

```diff
@@ -0,0 +1,34 @@
+contrib/avro/tests/python/pants_test/contrib/avro/tasks:avro_java_gen
```

Comment:
As described in PR description, we no longer special case contrib unit tests and they are now tied into the blacklist mechanism. This allows us to make more progress on getting them remoted.